kuzzleio/kourou
# instance:list and instance:kill commands

## What does this PR do?
First contribution to kourou and first use of ts, please be lenient.

Add the `kourou instance:list` and the `kourou instance:kill` commands

### How should this be manually tested?
  - Step 1 : Run some kuzzle instance from both 1 & 2 versions
  - Step 2 : `kourou instance:list` should display informations about your current running stacks
  - Step 3 : `kourou instance:kill -i stack-0_kuzzle_1` should kill the containers of the stack 0
  - Step 4 : `kourou instance:kill` should display a prompt to select the instance to kill
  - Step 5 : `kourou instance:kill --all` should kill all running instances
 

### Other changes
/

### Boyscout
Throw if prompt must appear using `kourou instance:logs` to select a kuzzle instance but no one is running
Tests for `instance:spawn`

### Screenshots
![image](https://user-images.githubusercontent.com/44427849/85537434-42878400-b614-11ea-863e-753cad0915dc.png)
